### PR TITLE
FIx the broken link for General Contribution Flow page

### PR DESCRIPTION
**Pull request by 75asu**
**Description**

This PR fixes the broken link for General Contribution Flow page

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit dd5a72a81a2b3abd9fadc8a1f618b9e2086b6aa6 at: https://631345ebe3237b0de3c19a3d--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 250904c34d0db11937491123530cf7ff473b447d at: https://6313545e7f1ade1a1d6d831d--meshery-docs-preview.netlify.app

**Comment by legendarykamal** [comment]
@measutosh 
1> Please make sure to file an issue before sending a PR for it.
2> I think the link works perfectly fine and redirects to the right page https://github.com/meshery/meshery/blob/master/docs/CONTRIBUTING-gitflow.md and its needed rightfully.

Theres no need of this change .

**Comment by legendarykamal** [comment]
closing this , you are welcome to contribute in other issues :)

**Comment by measutosh** [comment]
> @measutosh 1> Please make sure to file an issue before sending a PR for it. 2> I think the link works perfectly fine and redirects to the right page https://github.com/meshery/meshery/blob/master/docs/CONTRIBUTING-gitflow.md and its needed rightfully.
> 
> Theres no need of this change .

The earlier link leads to a 404 error page - project/contributing/CONTRIBUTING-gitflow.md

I because the page has been moved to a new place which is this -
/docs/CONTRIBUTING-gitflow.md).


I feel this new path should lead to the correct page. Please check this

**Comment by legendarykamal** [comment]
yes @measutosh sorry it does get to a 404 error page but you will have to make more changes as yours too doesnt get to the desired .

make it redirect to  /contributing-gitflow/ and see if it works :)

**Comment by measutosh** [comment]
> yes @measutosh sorry it does get to a 404 error page but you will have to make more changes as yours too doesnt get to the desired .
> 
> make it redirect to /contributing-gitflow/ and see if it works :)

Yes, I know. Don't know what's wrong with it
I found the actual path for the file but the dynamic url isn't working
Should I replace it with status url as in
Instead of "{base url}/path", should make it "https://meshery. Io/path"
Will that work out? Could suggest me the url

**Comment by l5io** [comment]
🚀 Preview for commit 250904c34d0db11937491123530cf7ff473b447d at: https://6313c29cc38f746023439f8c--meshery-docs-preview.netlify.app

File: docs/pages/project/contributing/contributing.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ Follow these steps and you'll be right at home.
 
 ## <a name="contributing">General Contribution Flow</a>
 
-To contribute to Meshery, please follow the fork-and-pull request workflow described [here]({{site.baseurl}}/project/contributing/CONTRIBUTING-gitflow.md).
+To contribute to Meshery, from creating a fork to creating pull request, please follow the basic fork-and-pull request workflow described [here]({{site.baseurl}}/docs/CONTRIBUTING-gitflow.md).
```
Comment (warunicorn19) [inline docs/pages/project/contributing/contributing.md:38]:
```suggestion
To contribute to Meshery, from creating a fork to creating pull request, please follow the basic fork-and-pull request workflow described [here]({{site.baseurl}}/project/contributing/contributing-gitflow).
```

**Comment by l5io** [comment]
🚀 Preview for commit d970b0c0b801911b3bad10f5369af8961f346e14 at: https://631664921693a93a4c058e6d--meshery-docs-preview.netlify.app

**Review by warunicorn19 — APPROVED** [review]
LGTM!

**Pull request merged**